fairer-iOS/fairer-iOS/Screens/OnboardingProfile/UIComponent/OnboardingProfileGroupCollectionViewCell.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  OnboardingProfileGroupCollectionViewCell.swift
+//  fairer-iOS
+//
+//  Created by 김유나 on 2022/09/19.
+//
+
+import UIKit
+
+import SnapKit
+
+final class OnboardingProfileGroupCollectionViewCell: BaseCollectionViewCell {
+    override var isSelected: Bool {
+        didSet {
+            DispatchQueue.main.async {
+                self.checkCircleView.isHidden = !self.isSelected
+            }
+        }
+    }
+    
+    // MARK: - property
+    
+    let profileImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = ImageLiterals.profileBlue3
+        return imageView
+    }()
+    private let checkCircleView: UIView = {
+        let view = UIView()
+        view.layer.borderWidth = 1
+        view.layer.borderColor = UIColor.blue.cgColor
+        view.isHidden = true
+        return view
+    }()
+    
+    // MARK: - lifecycle
+    
+    override func render() {
+        self.addSubview(profileImage)
+        profileImage.snp.makeConstraints {
+            $0.top.bottom.width.height.equalToSuperview()
```

Comment:
p3;

superview의 크기에 맞추고 싶은거라면
```suggestion
            $0.edges.equalToSuperview()
```
요렇게 할 수 있슴다 !